openshift/github-mirror-acceptance.yaml

```diff
@@ -4,40 +4,49 @@ kind: Template
 metadata:
   name: github-mirror-acceptance
 objects:
-- apiVersion: batch/v1
-  kind: Job
-  metadata:
-    name: github-mirror-acceptance-${IMAGE_TAG}
-  spec:
-    backoffLimit: 5
-    template:
-      spec:
-        restartPolicy: Never
-        containers:
-          - image: ${IMAGE}:${IMAGE_TAG}
-            imagePullPolicy: Always
-            name: github-mirror-acceptance
-            env:
-              - name: GITHUB_MIRROR_URL
-                value: ${GITHUB_MIRROR_URL}
-              - name: CLIENT_TOKEN
-                valueFrom:
-                  secretKeyRef:
-                    name: ${CLIENT_TOKEN_SECRET}
-                    key: CLIENT_TOKEN
-            command:
-              - make
-            args:
-              - accept
+  - apiVersion: v1
```

Comment:
any particular reason for the relinting in this PR? It's preferable that your changes are limited to what the PR  subject says.

Reply:
will revert the linting and submit a new PR